libs/deal-ticket/src/components/market-info/info-key-value-table.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  t,
+  addDecimalsFormatNumber,
+  formatNumberPercentage,
+  formatNumber,
+} from '@vegaprotocol/react-helpers';
+import {
+  KeyValueTableRow,
+  KeyValueTable,
+  Tooltip,
+} from '@vegaprotocol/ui-toolkit';
+import BigNumber from 'bignumber.js';
+import omit from 'lodash/omit';
+import startCase from 'lodash/startCase';
+import type { ReactNode } from 'react';
+import { tooltipMapping } from './tooltip-mapping';
+
+interface RowProps {
+  field: string;
+  value: any;
```

Comment:
chore (blocking): can we type this properly.

It looks like we aren't using generating types for the MARKET_INFO_QUERY. Can we do so please. Then all the types should flow nicely through this part of the app.

Reply:
https://github.com/vegaprotocol/frontend-monorepo/pull/1133/files#r961402382